kivy/uix/filechooser.py

```diff
@@ -512,6 +514,16 @@ class FileChooserController(RelativeLayout):
     .. versionadded:: 1.8.0
     '''
 
+    font_name = StringProperty(DEFAULT_FONT)
```

Comment:
I think `font` is a better name to use as that's what we use elsewhere in kivy, I believe. @tshirtman ?

Reply:
Label uses font_name at least, but I wouldn't be surprised if there were inconsistencies in this.